src/travel-details-screens/use-realtime-text.ts

```diff
@@ -29,36 +28,5 @@ export const useRealtimeText = (
     );
   }
 
-  const firstStop = estimatedCalls[0];
-
-  if (
-    firstStop?.quay?.name &&
-    firstStop.realtime &&
-    !isInThePast(firstStop.expectedDepartureTime)
-  ) {
-    return t(
-      DepartureDetailsTexts.noPassedStop(
-        firstStop.quay.name,
-        formatToClock(
-          firstStop.expectedDepartureTime,
-          language,
-          'floor',
-          debugShowSeconds,
-        ),
-      ),
-    );
-  }
-  const timeRepType = getTimeRepresentationType({
-    missingRealTime: !firstStop?.realtime,
-    aimedTime: firstStop?.aimedDepartureTime,
-    expectedTime: firstStop?.expectedDepartureTime,
-  });
-  switch (timeRepType) {
-    case 'no-significant-difference':
-      return t(DepartureDetailsTexts.onTime);
-    case 'significant-difference':
-      return t(DepartureDetailsTexts.notOnTime);
```

Comment:
It might that we need to remove `DepartureDetailsTexts.noontime` in the translations